packages/main/src/SegmentedButton.js

```diff
@@ -127,7 +127,7 @@ class SegmentedButton extends UI5Element {
 	}
 
 	onExitDOM() {
-		ResizeHandler.deregister(this.parentNode, this._handleResizeBound);
+		ResizeHandler.deregister(this, this._handleResizeBound);
```

Comment:
I think If we deregister on the "this"
```js
ResizeHandler.deregister(this, this._handleResizeBound);
```

it's best to register on "this" at the first place:
```js
ResizeHandler.register(this.parentNode, this._handleResizeBound);
```

Reply:
maybe check for the parentnode, and then derigister

Reply:
We will always have a parentNode on registering, but when deleting we won't have a parent element, so checking will give different results in both cases, this will mean mismatch between register and deregister. I've missed one of the methods and i think i will stick to the idea of this on both places.